efibuild.sh

```diff
@@ -437,12 +437,16 @@ fi
 
 if [ "$SKIP_BUILD" != "1" ]; then
   echo "Building..."
-  for arch in "${ARCHS[@]}" ; do
+  for i in "${!ARCHS[@]}" ; do
     for toolchain in "${TOOLCHAINS[@]}" ; do
       for target in "${TARGETS[@]}" ; do
         if [ "$MODE" = "" ] || [ "$MODE" = "$target" ]; then
-          echo "Building ${SELFPKG_DIR}/${SELFPKG}.dsc for $arch in $target with ${toolchain} and flags $BUILD_STRING ..."
-          buildme -a "$arch" -b "$target" -t "${toolchain}" -p "${SELFPKG_DIR}/${SELFPKG}.dsc" "${BUILD_ARGUMENTS[@]}" || abortbuild
+          echo "Building ${SELFPKG_DIR}/${SELFPKG}.dsc for ${ARCHS[i]} in $target with ${toolchain} and flags $BUILD_STRING ..."
```

Comment:
Add `${ARCHS_EXT[i]}` to `echo` maybe? Otherwise looks good.

Reply:
Good idea!